pages/linux/chroot.md

```diff
@@ -0,0 +1,23 @@
+# chroot
+
+> Run command or interactive shell with special root directory.
+
+- General Syntax
+
+`chroot [OPTION] NEWROOT {{COMMAND [ARG...]}}`
+
+- specify user and group (ID or name) to use
+
+`chroot −−userspec={{USER:GROUP}}`
+
+- specify supplementary groups as g1,g2,..,gN
```

Comment:
I'd update this to be just "specify supplementary groups" and then handle the formatting in the line below, like `chroot --groups={{group1,group2,...}}. I do however want someone else's opinion on this one.